src/EnergyPlus/HVACInterfaceManager.hh

```diff
@@ -69,16 +69,20 @@ namespace HVACInterfaceManager {
     constexpr int PrimaryRecirc(1);   // flow from Supply-outlet/Demand-inlet to Supply-inlet/demand-outlet
     constexpr int SecondaryRecirc(2); // flow from Supply-inlet/Demand-outlet to Supply-outlet/demand-inlet
 
-    constexpr int FlowTypeNotSet(9);
-    constexpr int ConstantFlow(10);
-    constexpr int VariableFlow(11);
+    enum class FlowType
+    {
+        Unassigned = -1,
+        FlowTypeNotSet,
+        ConstantFlow,
```

Comment:
Is `FlowTypeNotSet` the same as `Unassigned`? Also, don't need the word `Flow` or `FlowType` in the enumerations since they are part of the namespace.

Reply:
Thanks for your feedback. I'll make sure if Unassigned is overlapped or not, and not to use the name in enumeration which is part of namespace, in the future.